packages/create-keystone-app/example-projects/nuxt/index.js

```diff
@@ -0,0 +1,29 @@
+const { Keystone } = require('@keystone-alpha/keystone');
+const { Text } = require('@keystone-alpha/fields');
+const { GraphQLApp } = require('@keystone-alpha/app-graphql');
+const { AdminUIApp } = require('@keystone-alpha/app-admin-ui');
+const { NuxtApp } = require('@keystone-alpha/app-nuxt');
+
+const nuxtConfig = {
+  srcDir: 'src',
+  buildDir: 'dist',
+};
+
+const { MongooseAdapter: Adapter } = require('@keystone-alpha/adapter-mongoose');
```

Comment:
Doesn't this need a special comment like this? https://github.com/keystonejs/keystone-5/blob/eb84ddead30772fb86fff955e97697129a3848a7/packages/create-keystone-app/example-projects/starter/index.js#L6-L9